esm/interface/element.js

```diff
@@ -75,11 +75,19 @@ const isVoid = ({localName, ownerDocument}) => {
  * @implements globalThis.Element
  */
 export class Element extends ParentNode {
-  constructor(ownerDocument, localName) {
+  constructor(ownerDocument, localName, nsp) {
     super(ownerDocument, localName, ELEMENT_NODE);
     this[CLASS_LIST] = null;
     this[DATASET] = null;
     this[STYLE] = null;
+    this[NS] = nsp
```

Comment:
this is an implicit property of the element ... as this project only accepts SVG or non SVG elements, an `instanceof SVGElement` seems to be enough to return via lazy accessor the `nsp` ... adding extra fields to the class looks like a road to worse performance, so lazyness here would be much better, imho. What is the reason to attach something 90% of the use cases not needed?